.github/workflows/push-aptos-node.yaml

```diff
@@ -1,16 +1,34 @@
+# cspell:word localnet
 ---
+env:
+  # yamllint disable rule:key-ordering
+  DOCKER_IMAGE_PLATFORM: 'linux/amd64'
+  BIG_MONEY_GUY_PK: >-
+    0xa5e11183d0c23b8c49b40dd71bba7fa0992aed9b89c42934a1bb8f6e4ccc4ec4
+  PUBLISHER_PK: >-
```

Comment:
Just noting that for future use we might want to use `SECRET` and `PUBKEY` (both 6 letters, nice for indentation), and no ambiguity because technically private key and public key have the same acronym

Not asking to change, feel free to resolve unless you disagree

Reply:
I can use `SK` instead, that would probably be more clear than `PK`. But I generally avoid `public key` since each account's public key is not the same as its address and can have a fairly complex meaning on Aptos (multi-sig keys, multied25519 keys, scp256k1 keys, etc)

Reply:
Agreed that Aptos' schema confuses things even more, what with authentication keys holding more precedence than public keys. Mainly it's just that anywhere keypairs are involved this gotcha seems to keep coming up so I figured I'd flag here. Feel free to resolve

Reply:
Resolved in a separate PR to make sure it doesn't break other things: https://github.com/econia-labs/emojicoin-dot-fun/pull/235